src/EnergyPlus/PluginManager.cc

```diff
@@ -379,6 +411,82 @@ void PluginManager::setupOutputVariables([[maybe_unused]] EnergyPlusData &state)
 #endif
 }
 
+#if LINK_WITH_PYTHON
+void initPython(EnergyPlusData &state, fs::path const &pathToPythonPackages)
+{
+    PyStatus status;
```

Comment:
I do wish I had been able to use the PyConfig method a long time ago, but at the time I think I was still using 3.7 or earlier.  This is so much nicer nowadays.